collaborative/IOU/heat_index.ipynb

```diff
@@ -49,7 +51,7 @@
     "## Step 1: Select data\n",
     "\n",
     "### 1a) Grab location of interest by latitude and longitude\n",
-    "First we'll grab specific locations of interest, by using the latitude and longitude locations of a weather station, and provide code to input a custom lat-lon location. Furthermore, we will **not** be retrieving the actual station data that is bias-corrected to that station for this example. At present bias-corrected station data on the Analytics Engine only provides air temperature as a variable, and for Heat Index we must also have either dew point temperature (coming soon!) or relative humidity. So for the time being, we will retrieve **non-bias corrected** data at the location of interest."
+    "First we'll grab specific locations of interest, by using the latitude and longitudeconvert_to_local_timeions of a weather station, and provide code to input a custom lat-lon location. Furthermore, we will **not** be retrieving the actual station data that is bias-corrected to that station for this example. At present bias-corrected station data on the Analytics Engine only provides air temperature as a variable, and for Heat Index we must also have either dew point temperature (coming soon!) or relative humidity. So for the time being, we will retrieve **non-bias corrected** data at the location of interest."
```

Comment:
```suggestion
    "First we'll grab specific locations of interest, by using the latitude and longitude locations of a weather station, and provide code to input a custom lat-lon location. Furthermore, we will **not** be retrieving the actual station data that is bias-corrected to that station for this example. At present bias-corrected station data on the Analytics Engine only provides air temperature as a variable, and for Heat Index we must also have either dew point temperature (coming soon!) or relative humidity. So for the time being, we will retrieve **non-bias corrected** data at the location of interest."
```

Reply:
I'm not sure if you meant to include `convert_to_local_time` in this sentence differently, but I figured it's a typo as it is right now.

Reply:
This is a good catch - definitely a typo, and some overzealous copy & paste